apps/www/styles/index.css

```diff
@@ -20,6 +20,7 @@
 
 html {
   scroll-behavior: smooth;
+  width: fit-content;
```

Comment:
hey @hkaur008 - could you remove all other changes apart from this? 😄

Reply:
sure !